packages/ChangesReloaded-Tests.package/CRBrowserTest.class/instance/testClipChangeListSelectionIndexToValidValue.st

```diff
@@ -0,0 +1,11 @@
+accessing
+testClipChangeListSelectionIndexToValidValue
+
+	self assertCorrectClipOfChangeListSelectionIndexToLowerBound.
+	
+	self changesBrowser changeListSelectionIndex: 4.
+	self assert: self changesBrowser changeListSelectionIndex = 4.
+	self changesBrowser clipChangeListSelectionIndexToValidValue.
+	self assert: self changesBrowser changeListSelectionIndex = 4.
```

Comment:
since the other assertions are ausgelagert this should be also ausgelagert. sorry for the denglsich. could be called assertCorrectClipOfChangeListSelectionIndexToAlreadyValidIndex